packages/frameworks/src/types.ts

```diff
@@ -135,11 +135,11 @@ export interface Framework {
     /**
      * Default Build Command or a placeholder
      */
-    buildCommand: SettingValue;
+    buildCommand: Setting;
     /**
      * Default Development Command or a placeholder
      */
-    devCommand: SettingValue;
+    devCommand: Setting;
```

Comment:
This is causing TS to fail. Any reason why it was changed?

Reply:
Yes - but I updated this to just return `null` for the value because it shouldn't be defined here because it is project based